app/src/lib/git/diff.ts

```diff
@@ -454,6 +461,55 @@ async function getImageDiff(
   }
 }
 
+/**
+ * This function splits diff hunk into this format needed for `git lfs smudge`:
+ * `version https://git-lfs.github.com/spec/v1
+ *  oid sha256:[lfs file sha256]
+ *  size [lfs file size]`
+ * From diff hunk it can be get Added and/or Removed LFS text in format above
+ * and pass it to getLFSBlobImage to get image blob using `git lfs smudge`
+ */
+async function getLFSImageDiff(
+  repository: Repository,
+  file: FileChange,
+  hunk: DiffHunk
+): Promise<IImageDiff> {
+  let prev: string = ''
+  let curr: string = ''
+
+  for (let i = 0; i < hunk.lines.length; i++) {
+    const element = hunk.lines[i]
+    // strip first character for this Add/Delete line type
+    if (element.type === DiffLineType.Delete) {
+      // delete type line start with '-' and we need to skip this character
+      prev += element.text.substring('-'.length) + '\n'
+    }
+
+    if (element.type === DiffLineType.Add) {
+      // add type line start with '+' and we need to skip this character
+      curr += element.text.substring('+'.length) + '\n'
+    }
+  }
```

Comment:
```suggestion
    // strip first character ('+' or '-') for this Add/Delete line types
    if (element.type === DiffLineType.Delete) {
      prev += element.text.substring(1) + '\n'
    }

    if (element.type === DiffLineType.Add) {
      curr += element.text.substring(1) + '\n'
    }
  }
```

Nit: I think your comment speaks well enough as to not need to do a length check on a invariable length character.